education/urls.py

```diff
@@ -17,4 +18,5 @@
     path("lesson/create/", LessonCreateAPIView.as_view(), name="lesson_create"),
     path("lesson/delete/<int:pk>/", LessonDestroyAPIView.as_view(), name="lesson_delete"),
     path("lesson/update/<int:pk>/", LessonUpdateAPIView.as_view(), name="lesson_update"),
+    path("course/subscribe/", UpdateSubscriptionCourseAPIView.as_view(), name="subscribe_on_course"),
```

Comment:
В данном случае проще даже сделать так
```suggestion
    path(
        "course/<int:course_id>/subscribe/", 
        UpdateSubscriptionCourseAPIView.as_view(), 
        name="subscribe_on_course"
    ),
    
```
Тогда сериализатор вообще будет не нужен, а во вьюшке сделать так:
```python
def post(self, request, course_id):
    course = get_object_or_404(Course, id=course_id)
    user = request.user
    ...
```